Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -33,7 +33,7 @@ class DefaultLocationService: LocationService {
     }
 
     func changeLocationEngineResolution(resolution: Resolution) {
-        // TODO: Implement method
+        locationManager.systemLocationManager.desiredAccuracy = resolution.accuracy.toCoreLocationAccuracy()
```

Comment:
Awesome that you don't have to rewrite the whole location manager to do that :muscle: However, two things are missing (desired interval and minimum displacement), look at how it's implemented in the [Android code](https://github.com/ably/ably-asset-tracking-android/blob/main/publishing-sdk/src/main/java/com/ably/tracking/publisher/locationengine/MappingFunctions.kt#L7-L11) :wink:

Reply:
Ok, I forgot about `distanceFilter` I'll add it soon, but there is no way to specify something like `time intervals` between checking locations in CLLocationManager

Reply:
Update here 9422f28c264115162a959e39403be54f722a2e5d

Reply:
It looks better now, thanks :bow: It's a shame that we can't modify the time interval :disappointed: I think it's worth mentioning somewhere in the docs or API as this might affect some of users expectations, WDYT @AndyNicks ? :thinking:

Reply:
Yes, it is definitely has to be mentioned in the documentation. I think it has to me bentioned in README as well

Reply:
I'm not sure if I need to mention it in README.md @AndyNicks, actually, there is no mention of `Resolution` so I would leave it as it is. Who I should inform in case of documentation update?

Reply:
AFAIK there's already an issue created for this problem, therefore, I'm resolving this conversation :wink: